app/controllers/submissions_controller.rb

```diff
@@ -120,6 +136,28 @@ def destroy
                                                    @submission.assessment)) && return
   end
 
+  action_auth_level :destroyBatch, :instructor
+  def destroyBatch
+    submission_ids = params[:submission_ids]
+    submissions = submission_ids.map { |sid| @assessment.submissions.find_by(id: sid) }
```

Comment:
if you're just trying to get the submission objects associated with `submission_ids`, you can just do `Submission.where(id: submission_ids)`. In general, you should use the built in SQL ODM methods over writing custom .map, .each, etc. behavior whenever possible.

Reply:
Fixed.